ring/ring.go

```diff
@@ -677,10 +685,6 @@ func (r *Ring) updateRingMetrics(compareResult CompareResult) {
 		r.oldestTimestampGaugeVec.WithLabelValues(state).Set(float64(timestamp))
 	}
 
-	if compareResult == EqualButStatesAndTimestamps {
```

Comment:
This check used to be useful when we did more updates in `updateRingMetrics` after this point, but now we only update single gauge, so it seems useless.